src/index.html

```diff
@@ -1,14 +1,21 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Snake</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="blocks">
+      <div class="block block--1"><span class="number">1</span></div>
+      <div class="block block--2"><span class="number">2</span></div>
+      <div class="block block--3"><span class="number">3</span></div>
+      <div class="block block--4"><span class="number">4</span></div>
+      <div class="block block--5"><span class="number">5</span></div>
+      <div class="block block--6"><span class="number">6</span></div>
```

Comment:
```suggestion
      <div class="block"><span class="number">1</span></div>
      <div class="block"><span class="number">2</span></div>
      <div class="block"><span class="number">3</span></div>
      <div class="block"><span class="number">4</span></div>
      <div class="block"><span class="number">5</span></div>
      <div class="block"><span class="number">6</span></div>
```
Используй nth-child вместо дополнительных классов